packages/scaffolding-cli/templates/src/csr/config/env.js

```diff
@@ -72,6 +72,7 @@ function getClientEnvironment(publicUrl) {
                 // Useful for determining whether we’re running in production mode.
                 // Most importantly, it switches React into the correct mode.
                 NODE_ENV: process.env.NODE_ENV || "development",
+                API_BASE: process.env.API_BASE || "http://localhost:3000",
```

Comment:
API_BASE should be determined by NODE_ENV correct? Rather than a default?

Reply:
Yes. But this is to help run in the localhost env by default. I will update the readmes.